src/directLine.ts

```diff
@@ -44,6 +44,13 @@ export interface Media {
     thumbnailUrl?: string
 }
 
+export interface UnknownMedia{
+    contentType: string,
+    contentUrl: string,
+    name?: string,
```

Comment:
Both parameters (name and thumbnailUrl) are coming from the SDK and are set up as optional. 

Reference:
https://github.com/Microsoft/BotBuilder/blob/3a98a6b3d15962a57b5454bfb3f730d3588de3ef/CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/Attachment.cs